sidechain/sidechain-storage/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "its-sidechain-storage"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Version and edition should match the other crates. I think we're at `0.8.0` and edition is `2018`. Check in one of the other crates to be sure.